SyntaxError: invalid escape sequence \s

When running tests as part of the Debian build process for astroquery 0.4.4, I get several occurrences of `SyntaxError: invalid escape sequence \s`.

Here is the log:
```
============================= test session starts ==============================
platform linux -- Python 3.9.9, pytest-6.2.5, py-1.10.0, pluggy-0.13.0

Running tests with Astropy version 4.3.1.
Running tests in .pybuild/cpython3_3.9_astroquery/build.

Date: 2021-11-22T18:50:22

Platform: Linux-5.10.0-9-amd64-x86_64-with-glibc2.32

Executable: /usr/bin/python3.9

Full Python Version: 
3.9.9 (main, Nov 16 2021, 10:24:31) 
[GCC 11.2.0]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Package versions: 
Numpy: 1.21.4
Scipy: 1.7.1
Matplotlib: 3.5.0
h5py: not available
Pandas: not available

Using Astropy options: remote_data: none.

rootdir: /build/astroquery-0.4.4+dfsg, configfile: setup.cfg
plugins: mock-3.6.1, hypothesis-5.43.3, openfiles-0.5.0, filter-subpackage-0.1.1, doctestplus-0.11.1, cov-3.0.0, arraydiff-0.3, astropy-header-0.1.2, remotedata-0.3.2
collected 1419 items / 8 errors / 1411 selected

==================================== ERRORS ====================================
_ ERROR collecting .pybuild/cpython3_3.9_astroquery/build/astroquery/ipac/ned/__init__.py _
astroquery/ipac/ned/__init__.py:84: in <module>
    from .core import Ned, NedClass
E     File "/build/astroquery-0.4.4+dfsg/.pybuild/cpython3_3.9_astroquery/build/astroquery/ipac/ned/core.py", line 521
E       f'<a\s+href\s*?="?\s*?(.+?{extensions[file_format]})"?\s*?>\s*?(?:Retrieve|{names[file_format]})</a>',
E       ^
E   SyntaxError: invalid escape sequence \s
```

--- a/astroquery/ipac/ned/core.py
+++ b/astroquery/ipac/ned/core.py
@@ -518,7 +518,7 @@ class NedClass(BaseQuery):
                  'VO-table': 'VOTable'}
 
         pattern = re.compile(
-            f'<a\s+href\s*?="?\s*?(.+?{extensions[file_format]})"?\s*?>\s*?(?:Retrieve|{names[file_format]})</a>',
+            fr'<a\s+href\s*?="?\s*?(.+?{extensions[file_format]})"?\s*?>\s*?(?:Retrieve|{names[file_format]})</a>',
             re.IGNORECASE)
         matched_urls = pattern.findall(html_in)
         url_list = [base_url + img_url for img_url in matched_urls]

